src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -369,6 +369,31 @@ def post(self, request: Request, organization) -> Response:
             )
             return Response({"detail": ERR_RATE_LIMITED}, status=429)
 
+        is_member = not request.access.has_scope("member:admin") and (
+            request.access.has_scope("member:invite")
+        )
+        disable_invites_to_other_teams = (
+            not organization.flags.allow_joinleave and not organization.flags.disable_member_invite
+        )
```

Comment:
i think this should be `can_invite_to_member_teams`, this looks like if open membership is off and member invite is on? i had to spend a bit of time grokking what line 382 was doing -- negatives are also hard to reason about